sdk/python/kfp/__init__.py

```diff
@@ -25,3 +25,4 @@
 from ._config import *
 from ._local_client import LocalClient
 from ._runners import *
+from ._credentials import *
```

Comment:
The credentials seem Client-specific, so I'm not sure we should import them in the root kfp package.

Reply:
Hi @Ark-kun, I just saw this. Do you have some specific structure in mind?